package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.5",
```

Comment:
What is @emotion/react and @emotion/styles for?

Reply:
It came with react